### PR TITLE
MULTIARCH-3240 Update TP and Depr tables for IBM Power and IBM Z

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/MULTIARCH-3240

Link to docs preview: 
- [Deprecated Features](https://57256--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-deprecated-removed-features)
- [Technology Preview](https://57256--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-technology-preview)

QE review:
- [x] QE has approved this change.

Additional information:

@alishaIBM

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -483,6 +483,20 @@ In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack
 [id="ocp-4-13-removed-features"]
 === Removed features
 
+[id="ocp-4-13-z13-power8-x86v1-remove"]
+==== Specific hardware models on `ppc64le`, `s390x`, and `x86_64` v1 CPU architectures are removed
+
+In {product-title} 4.13, support for {op-system} functionality is removed for the following deprecated hardware models:
+
+* IBM POWER8 all models (ppc64le)
```
Comment (opayne1) [inline release_notes/ocp-4-13-release-notes.adoc:491]: Should this and the following be in backticks? 

```suggestion
* IBM POWER8 all models (`ppc64le`)
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -483,6 +483,20 @@ In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack
...
+* IBM POWER9 AC922 (ppc64le)
+* IBM POWER9 IC922 (ppc64le)
+* IBM POWER9 LC922 (ppc64le)
+* IBM z13 all models (s390x)
```
Comment (opayne1) [inline release_notes/ocp-4-13-release-notes.adoc:495]:
```suggestion
* IBM z13 all models (`s390x`)
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -483,6 +483,20 @@ In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack
...
+* {linuxoneProductName} Emperor (s390x)
+* {linuxoneProductName} Rockhopper (s390x)
+* AMD64 (x86_64) v1 CPU
```
Comment (opayne1) [inline release_notes/ocp-4-13-release-notes.adoc:498]:
```suggestion
* AMD64 (`x86_64`) v1 CPU
```

**Review by opayne1 — COMMENTED** [review]
A few tiny nits, but otherwise lgtm! FYI @JoeAldinger

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (SNiemann15) [reply release_notes/ocp-4-13-release-notes.adoc:491]: Good idea 🥇

**Comment by lsmcfadden** [comment]
@SNiemann15 
@alishaIBM got confirmation from our Power naming focal that the Power models IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 mentioned in this PR are to be written as IBM Power AC922, IBM Power IC922, and IBM Power LC922.  Can you update or how should this be handled?

**Comment by alishaIBM** [comment]
> @SNiemann15
@alishaIBM got confirmation from our Power naming focal that the Power models IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 mentioned in this PR are to be written as IBM Power AC922, IBM Power IC922, and IBM Power LC922. Can you update or how should this be handled?

@SNiemann15 I agree to the above mentioned change. Also `IBM POWER8` is to be written as `IBM Power8`. Request you to update the PR with the above changes. Thanks.

**Comment by JoeAldinger** [comment]
I pulled all of these changes into #58100 . @SNiemann15 can you double check and ack the PR to make sure all looks good, please.

**Comment by SNiemann15** [comment]
Closing this PR as updates haven been pulled into https://github.com/openshift/openshift-docs/pull/58100

**Pull request closed without merging**